### PR TITLE
Fixing the examples

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: demos/dpc/download.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    if unzip_path is None:
+        unzip_path = os.path.join(current_folder, 'SOFC')
+    if not os.path.exists(unzip_path):
+        z = zipfile.ZipFile(download_path)
+        print("extracting to --> %s" % unzip_path)
+        z.extractall(path=unzip_path)
+
+
+if __name__ == "__main__":
+    run()
```
Comment (sameera2004) [inline demos/dpc/download.py]: demos/dpc/dpc_demo.ipynb in this PR has erros

**Comment by sameera2004** [comment]
@ericdill you have added `demos/speckle/speckle-plotting.ipynb` in this PR, but the `demos/dpc/dpc_demo.ipynb` is not working, (if I merge it, it will overite working `demos/dpc/dpc_demo.ipynb` earlier version).
 Can you please fix it then I will be able to merge this PR?

File: demos/dpc/download.py (hunk 1 above)
Comment (ericdill) [inline demos/dpc/download.py]: Thank you for pointing that out. I am working on fixing it now

**Comment by ericdill** [comment]
@sameera2004 this should be good to go

**Comment by ericdill** [comment]
https://github.com/NSLS-II/wishlist/issues/32 can be closed once this is merged

File: demos/prepare_for_docker.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import sys
+import os
+import importlib
+import subprocess
+import zipfile
+import zipfile
```
Comment (sameera2004) [inline demos/prepare_for_docker.py]: zipfile imported twice

Comment (ericdill) [inline demos/prepare_for_docker.py]: ah, good catch

**Pull request merged**